shared/js/content-scripts/fb-surrogate-xray.js

```diff
@@ -16,7 +16,8 @@
     let sdkPatterns = [/connect.facebook.net\/[a-zA-Z_]+\/(sdk|all).js/]
     // Request list of rules to capture from the extension
     chrome.runtime.sendMessage({
-        getSocialSurrogateRules: entity
+        messageType: 'getSocialSurrogateRules',
+        options: entity
```

Comment:
why change of a format?

Reply:
Same as above, it eases the handling in the `onMessage` handler without it we would have to check *all* the properties of the message and see if it aligns with an `export` name which seemed very fragile.

I moved to the `export` because having an `async` handler when sometimes there isn't a response meant it might never complete which is obviously non ideal. The change wasn't required but the alternative made the same kinda size of change in the `onMessage` handler at least.

For example some messages look like: `{setListContents: true, value: {}}`

If we added a `export function value() {}` we'd end up calling the other method.